pkg/tags/resource_tags.go

```diff
@@ -0,0 +1,52 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package tags
+
+// ResourceTags represents the AWS tags which will be added to the AWS resource.
+// Inside aws-sdk-go, Tags are represented using multiple types, Ex: map of
+// string, list of structs etc...
+// ResourceTags type will be used as a hub/mediator to merge tags represented
+// using different types.
+type ResourceTags struct {
```

Comment:
How about just calling this `Tags`? I don't think there is value in having ResourceTags versus some other type of Tags... it's all just a map[string]string anyway :)